README.md

```diff
@@ -21,7 +21,7 @@
 
     ```
     $ pip install -r requirements/common.txt
-    $ pip install -r requirements/dev.txt
+    $ pip install -r requirements/dev.  txt
```

Comment:
I'm not really sure. Do you know which one is the correct version? (sorry for the late response)